presto-main/src/test/java/com/facebook/presto/operator/scalar/TestColorFunctions.java

```diff
@@ -130,6 +131,7 @@ public void testRenderDouble()
             throws Exception
     {
         assertEquals(render(1234.5678, color(toSlice("red"))), toSlice("\u001b[38;5;1m1234.5678\u001b[0m"));
+        assertEquals(render(1234.5678f, color(toSlice("red"))), toSlice(format("\u001b[38;5;1m%s\u001b[0m", (double) 1234.5678f)));
```

Comment:
No I just wanted to test color functions with floats, I can add them in a different commit